hydropandas/extensions/plots.py

```diff
@@ -307,6 +307,247 @@ def interactive_map(
+                               marker=(3, 0, 0),
+                               color='blue',
+                               alpha=0.5,
+                               label='95% observation',
+                               )
+            ax_section.scatter(plot_x[counter],
+                               plot_df.median(),
+                               section_markersize,
+                               marker=(4, 0, 90),
+                               color='green',
+                               alpha=0.5,
+                               label='median',
+                               )
+            ax_section.scatter(plot_x[counter],
+                               plot_df.quantile(q=0.05),
+                               section_markersize,
+                               marker=(3, 0, 180),
+                               color='red',
+                               alpha=0.5,
+                               label='5% observation',
+                               )
+
+        logger.info(f"created sectionplot -> {name}")
+
+        # layout
+        if section_label_x is None:
+            # use name as xtick and rotate
+            ax_section.set_xticks(plot_x,
+                                  self._obj.index,
+                                  rotation='vertical',
+                                  fontsize='small',
+                                  )
+        else:
+            ax_section.set_xlabel(section_label_x)
+
+        # rotate labels on observation axis
+        ax_obs.set_xticks(ax_obs.get_xticks(),
+                          ax_obs.get_xticklabels(),
+                          rotation='vertical',
+                          fontsize='small',
+                          )
+
+        if ylabel == 'auto':
+            # has collection uniform unit?
+            if len(self._obj.unit.unique()) == 1:
+                ylabel = self._obj.unit.unique()[0]
+            else:
+                logger.error(f"Collection has more than one unit. Plot has both on one y-axis.")
+                ylabel = ', '.join(map(str, self._obj.unit.unique()))
+            for ax in axes:
+                ax.set_ylabel(ylabel)
+        else:
+            try:
+                ax_section.set_ylabel(ylabel[0])
+                ax_obs.set_ylabel(ylabel[1])
+            except:
+                logger.error(f"Invalid value for ylabel {ylabel}. Plot has no ylabels.")
+            
+            
+        # add layout to both plots
+        for ax in axes:
+            ax.grid()
+
+            # simple legend
+            handles, labels = ax.get_legend_handles_labels()
+            by_label = dict(zip(labels, handles))
+            ax.legend(by_label.values(), by_label.keys())
+
+        fig.suptitle(self._obj.name)
+
+        if fn_save is not None:
+            try:
+                fig.savefig(fn_save, dpi=250, bbox_inches='tight',)
+            except:
+                logger.error(f"Save of figure {o.name} failed ({fn_save}).")
```

Comment:
variable `o` here does not exist, should be `self._obj`?

Reply:
Solved in update.